src/index.js

```diff
@@ -265,14 +265,14 @@ function unregisterNode (host, node, func) {
   }
 }
 
-function addNodeToNode (host, node, insertBefore) {
-  registerNode(host, node, insertBefore, function (eachNode) {
+function addNodeToNode(host, node, insertBefore) {
+  registerNode(host, node, insertBefore, (eachNode) => {
```

Comment:
parens around eachNode isn't technically required here - is this is the style we want? just double checking :)